docs/dataset/advanced-usage/extract-metadata-text.md

```diff
@@ -0,0 +1,274 @@
+---
+icon: kolena/comparison-16
+---
+
+# :kolena-comparison-20: Text Metadata Hydration
+
+This guide outlines how to configure the extraction of metadata from text data on Kolena. Follow the steps below
+to get started with Automatic Metadata Hydration for Text.
+
+## Configuring Metadata Extraction
+
+**Navigate to Dataset Details**: Scroll down to the "Details" page of your dataset.
+
+<figure markdown>
+![Navigating to Configuration](../../assets/images/navigate-to-text-extraction-config.gif)
+<figcaption>Navigating to Metadata Configuration</figcaption>
+</figure>
+
+**Select Text Fields and Properties**: Identify and select the text fields from your dataset that you want to analyze.
```

Comment:
As these are the steps to run the extraction, how about we use a numbered list instead?

We can also hide the details under a collapsible section.